include/openspace/scene/scenegraphnode.h

```diff
@@ -139,6 +139,8 @@ class SceneGraphNode : public properties::PropertyOwner {
     double reachFactor() const;
     double approachFactor() const;
 
+    bool isDirectlyTouchable() const;
```

Comment:
How about a more generic name, even though we only use it for touch right now?    Like `supportsDirectInteraction` instead?  To not tie it to the touch interaction too much

Reply:
Sure, but the description should still mention touch, for now, I think. As it currently IS very tied to the touch interaction, and I want that to be clear to the users. It's not going to be a relevant property for any user not using touch interaction, a of now at least

Reply:
Agree that the description should mention touch;  that we can change later once the value is applied to other interaction forms, too